Sources/Apodini/Semantic Model Builder/REST/RESTEndpointHandler.swift

```diff
@@ -6,7 +6,7 @@ import Foundation
 @_implementationOnly import Vapor
 import protocol FluentKit.Database
 
-extension Vapor.Request: ExporterRequest, WithEventLoop {}
+extension Vapor.Request: ExporterRequest, WithEventLoop, Reducible {}
```

Comment:
Is this up to date with the current develop?
If so I would leave pretty much the same comment as on https://github.com/Apodini/Apodini/pull/102#discussion_r552575194.
But it seems like it just got lost while rebasing(?)

Reply:
The automatic conformance was not lost, but the redundant conformances were added back in. Should be fixed now. Thanks!